src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  margin: 0 auto;
+  display: grid;
+
+  padding: 50px 40px;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 46px;
```

Comment:
The gap property has different horizontal and vertical values (48px 46px). It's usually a good practice to keep these values consistent unless there is a specific design requirement dictating otherwise. If the design does not require different values, consider using the same value for both to maintain consistency.